modules/api/src/main/java/org/apache/ignite/configuration/schemas/network/NetworkConfigurationSchema.java

```diff
@@ -23,18 +23,20 @@
 import org.apache.ignite.configuration.annotation.Value;
 import org.apache.ignite.configuration.storage.ConfigurationType;
 
-/**
- * Configuration schema for network endpoint subtree.
- */
 @ConfigurationRoot(rootName = "network", type = ConfigurationType.LOCAL)
 public class NetworkConfigurationSchema {
-    /** */
+    /** Uniq local node name. */
+    @Value(hasDefault = true)
+    public String name = "";
```

Comment:
As you've told let's move it to LocalConfiguation. 
Do you mean that "" is a default name? If true it seems to be not the best one to use.

Reply:
I haven't a way how to make a value which would a composition of another fields.
The reason of this value is we cannot to assign null.